asQ/allatonce/form.py

```diff
@@ -145,7 +145,7 @@ def assemble(self, func=None, tensor=None):
         self.aaofunc.update_time_halos()
 
         # assembly stage
-        fd.assemble(self.form, tensor=self.F.function)
+        fd.assemble(self.form, tensor=self.F.function).riesz_representation(riesz_map='l2')
```

Comment:
I'm confused about a couple of things here:

1. is it still valid to give a `Function` as the `tensor` argument if `assemble` is now meant to return a `Cofunction`?

1. Is sticking the call to `riesz_representation` on the end of the `assemble` call making use of the fact that the return value of `assemble` is the `tensor` argument (if given)?
If so then I think calling `riesz_representation` on a seperate line would be clearer.

1. Isn't the Riesz representer just being discarded here? I don't _think_ the `riesz_representer` method changes the object its called on, it just returns a value, which we aren't assigning to a variable.

Reply:
This call to `riesz_representation` can be removed now I think.